code/game/objects/items/weapons/shards.dm

```diff
@@ -87,6 +87,7 @@
 				H.Weaken(3)
 				if(affecting.take_damage(5, 0))
 					H.UpdateDamageIcon()
+					affecting.embed(src)
```

Comment:
Oh, uh, well. This should probably be its own PR; this doesn't really have anything to do with LINDA, at all.